tst/Object.ts

```diff
@@ -807,7 +807,7 @@ checks([
 ])
 
 // ---------------------------------------------------------------------------------------
-// OPTIONAL
+// OPTIONAL & PARTIAL
```

Comment:
You must create a separate bundle of tests for `Partial`, does not need to be crazy, but must at least reflect `deep` and `flat` usages.